include/os/windows/spl/sys/misc.h

```diff
@@ -1,22 +1,19 @@
 /*
- * CDDL HEADER START
+ *  The SPL is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License as published by the
+ *  Free Software Foundation; either version 2 of the License, or (at your
+ *  option) any later version.
```

Comment:
This isn't going to work. CDDL is used in the most of the source code is incompatible with GPL. If you really want to use GPL, it may be a dual license, e.g. `CDDL or (at your option) GPL`.

Reply:
I understand what you're saying but I actually created the file using a licence from another file as a template.
But checking while doing this I was shocked to learn the spl is appatently gpl licensed. see here https://github.com/openzfsonwindows/openzfs/blob/windows/include/os/windows/spl/spl-debug.h
So I think this file I made should be GPL like the rest of the spl.

Reply:
Oh, I'm sorry, my bad 🤦🏻‍♂️.

Reply:
no, no its okay. 
I had no idea until 2 hours ago, just like you.

Reply:
Not sure what to say here. It's a Windows file, nothing to do with Linux. There are 74 other files with same license (grep, grep), and finally, the file is empty.

Reply:
ah late to party, ignore me